Tasks/PublishPackageTask.cs

```diff
@@ -58,11 +58,14 @@ public override async Task RunAsync(BuildContext context)
         projectData = projectData.Replace("{LicencePath}", context.PackContext.LicensePath);
 
         if (context.PackContext.LicensePath.EndsWith(".txt"))
-            projectData = projectData.Replace("{LicenceName}", "LICENSE.txt");
+            projectData = projectData.Replace("{LicenceName}", "LICENSE.txt").Replace ("{LicencePackagePath}", "LICENSE.txt");
```

Comment:
You have a space character between Replace and `(`.

Same in the places bellow.

Reply:
Sorry, my editor is setup for mono style code by defualt, and there isn't an .editorconfig.